Branch-SDK/src/main/java/io/branch/referral/BranchUtil.java

```diff
@@ -47,6 +48,11 @@ static void shutDown() {
      * false if "io.branch.sdk.TestMode" is not added in the manifest or String res.
      */
     static boolean checkTestMode(Context context) {
+        BranchJsonConfig jsonConfig = new BranchJsonConfig(context);
```

Comment:
Wanted to add a static `BranchJsonConfig` instance to this class so we don't need to instantiate it on every class method. Unfortunately instantiating on `Branch` constructor didn't work because this method needs to be called before.
Is there any other safe place where we can instantiate it in terms of app lifecycle?

Reply:
I would guess you can do something in `Branch.getAutoInstance` using the Application Context passed in there. Another possibility is just process it once there and store the results instead of storing an instance of `BranchJsonConfig`.